test-commons/src/main/scala/za/co/absa/spline/scalatest/ConditionalIgnore.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.scalatest
+
+import org.scalatest.{Ignore, Tag}
+import org.apache.spark.SPARK_VERSION
+
+object ConditionalTestIgnore {
+
+  private def ignoreTestBySparkVersion(condition: String => Boolean): Tag = {
```

Comment:
The method name "ignore..." implies that something should be ignored when the `condition` holds. The implementation does the opposite thing - returns `@Ignore` tag when `condition` doesn't hold.

Also why does the `condition` function take a Spark version argument, considering the fact that one is a constant? Can't it be inlined?

Reply:
1. I was not able to find good opposite from "ignore"

Reply:
what if we simply rename it to, say `ignoreUnless(...)`, so that the name fits the signature?
Also what about making independent from the Spark version specifically? If you change the condition type to just `condition: => Boolean` you could use it in test directly like this:
```scala
it("should bla-bla-bla", ignoreUnless(!SPARK_VERSION.startsWith("2.2")) {
  ...
}
```
Or even better, to get rid of negations we could use another method name (might be an additional method delegating to the first one) - `ignoreWhen(SPARK_VERSION.startsWith("2.2"))`